imap_processing/codice/codice_l1a.py

```diff
@@ -118,9 +124,9 @@ def decompress_data(self, science_values: list[str]) -> None:
 
         Parameters
         ----------
-        science_values : list[str]
-            A list of byte strings representing the science values of the data
-            for each packet.
+        science_values : list[str] | str
+            A list of (or a single) byte string(s) representing the science
+            values of the data for each packet.
```

Comment:
Would it be easier to wrap the single i-alirt string in a list before this so you don't have to account for both cases? It'd just be a one-element list.

Reply:
Upon further examination I actually had this wrong. Even in my corner case of I-ALiRT data this is a `list[str]`, so thanks for commenting here!